Algorithm.Python/Alphas/GlobalEquityMeanReversionIBSAlpha.py

```diff
@@ -43,78 +42,71 @@
 # or <a href="https://github.com/QuantConnect/Lean/blob/master/Algorithm.Python/Alphas/GlobalEquityMeanReversionIBSAlpha.py">Python</a>.
 # 
 
-class GlobalEquityMeanReversionIBSAlphaAlgorithm(QCAlgorithmFramework):
+class GlobalEquityMeanReversionIBSAlpha(QCAlgorithmFramework):
 
     def Initialize(self):
 
         self.SetStartDate(2018, 1, 1)
 
         self.SetCash(100000)
-        
+
         # Set zero transaction fees
         self.SetSecurityInitializer(lambda security: security.SetFeeModel(ConstantFeeModel(0)))
-        
+
         # Global Equity ETF tickers
         tickers = ["ECH","EEM","EFA","EPHE","EPP","EWA","EWC","EWG",
                    "EWH","EWI","EWJ","EWL","EWM","EWM","EWO","EWP",
                    "EWQ","EWS","EWT","EWU","EWY","EWZ","EZA","FXI",
-                   "GXG","IDX","ILF","EWM","QQQ","RSX","SPY","THD"]           
-                   
+                   "GXG","IDX","ILF","EWM","QQQ","RSX","SPY","THD"]
+
         symbols = [Symbol.Create(ticker, SecurityType.Equity, Market.USA) for ticker in tickers]
-        
+
         # Manually curated universe
         self.UniverseSettings.Resolution = Resolution.Daily
         self.SetUniverseSelection(ManualUniverseSelectionModel(symbols))
-            
+
         # Use GlobalEquityMeanReversionAlphaModel to establish insights
         self.SetAlpha(MeanReversionIBSAlphaModel())
 
         # Equally weigh securities in portfolio, based on insights
         self.SetPortfolioConstruction(EqualWeightingPortfolioConstructionModel())
-        
-        ## Set immediate execution
-        self.SetExecution(ImmediateExecutionModel())
```

Comment:
Please leave default model changes in the examples, we want them for documentation.